src/tests/ftest/launch.py

```diff
@@ -2666,7 +2667,8 @@ def process(self, test, repeat, stop_daos, archive, rename, jenkinslog, core_fil
                     continue
                 return_code |= self._archive_files(
                     summary, data["hosts"].copy(), data["source"], data["pattern"],
-                    data["destination"], data["depth"], threshold, data["timeout"])
+                    data["destination"], data["depth"], test.yaml_info["test_log_threshold"],
```

Comment:
We still need to use the normal log size threshold if the test does not override it:
```suggestion
                    data["destination"], data["depth"],
                    test.yaml_info["test_log_threshold"] or threshold,
```

Reply:
ah,  thanks. @phender